test/Balance/SConscript

```diff
@@ -1,41 +1,43 @@
 Import('env')
 Import('parallel_run')
+Import('parallel_arg')
 Import('serial_run')
 Import('ip_charm')
-
 Import('bin_path')
 Import('test_path')
-
+import os
 #----------------------------------------------------------------------
 # defines
 #----------------------------------------------------------------------
 
 env['CPIN'] = 'touch parameters.out; mv parameters.out ${TARGET}.in'
 env['RMIN'] = 'rm -f parameters.out'
+env['clocal_cmd'] = '++local'
```

Comment:
More generally, it may make more sense to simply skip these tests on circleci, rather than adding the `++local` argument, since this explicitly is checking on load-balancing between processes. I think that this is how these tests were handled prior to the testing overhaul (but I may be mistaken).

If doing this would be difficult and time-consuming, it would be fine with me if you just leave things as they stand and open an Issue mentioning that this should be addressed.

Reply:
I think I will open an issue